src/pages/Account/Venue/VenueMapEdition/PartyMapContainer.tsx

```diff
@@ -1,24 +1,24 @@
 import React, { useMemo } from "react";
+import { Room } from "types/rooms";
 
 import { ExtractProps } from "types/utility";
-import { PartyMapVenue } from "types/venues";
 
 import { Container, SubVenueIconMap } from "./Container";
 
 type PartyMapContainerProps = Omit<
   ExtractProps<typeof Container>,
   "otherIcons"
 > & {
-  venue: PartyMapVenue;
+  rooms: Room[];
   currentRoomIndex?: number;
 };
 
 export const PartyMapContainer: React.FC<PartyMapContainerProps> = (props) => {
```

Comment:
Since you've removed `PartyMapVenue`, there is seemingly nothing that makes this a `PartyMapContainer` anymore? Can we rename this to something that better represents it's purpose? I haven't looked too deeply what it does/etc, but off the top of my head, potentially `RoomsContainer` or similar?

```suggestion
export const RoomsContainer: React.FC<RoomsContainerProps> = (props) => {
```